test/cypress/integration/printLogsFail.spec.js

```diff
@@ -0,0 +1,7 @@
+describe('Print Logs Fail.', () => {
+
+  it('Print Logs Fail', () => {
```

Comment:
You need to have two it's in a file. Also have the first one fail and the last one succeed. You can only that way test what I am referring to.